packages/ckeditor5-image/src/imagestyle/imagestyleui.js

```diff
@@ -216,3 +220,25 @@ function translateStyles( styles, titles ) {
 function getUIComponentName( name ) {
 	return `imageStyle:${ name }`;
 }
+
+// Returns the image style definition name without the `imageStyle:` prefix.
+//
+// @param {String} name
+// @returns {String}
+function getComponentDefinitionName( name ) {
+	return name.replace( 'imageStyle:', '' );
+}
+
+// TODO
+function normalizeDropdownConfig( config ) {
+	try {
+		return {
+			items: config.items.map( getComponentDefinitionName ),
```

Comment:
Let's use original button names so they are the same as if they were directly used in the toolbar (here to avoid confusion).